### PR TITLE
Oh My Posh

### PR DESCRIPTION
**Pull request by libewa**
closes https://github.com/Botspot/pi-apps/issues/2346

### CODE REVIEW

File: apps/Oh My Posh/install-32 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+wget -O /tmp/oh-my-posh https://github.com/JanDeDobbeleer/oh-my-posh/releases/latest/download/posh-linux-arm
```
Comment (theofficialgman) [inline apps/Oh My Posh/install-32:3]: I would still like this to be versioned (use the format like shown in the example install script from the documentation https://pi-apps.io/wiki/development/Creating-an-app/#making-an-install-script )

You should additionally add an auto-updater so that pi-apps can keep the users install of Oh My Posh up to date. I don't have great documentation written for how to do so but this commit should be a good start showing how to obtain the version number from github releases and how pi-apps expects to be able to update the version number of the application automatically https://github.com/Botspot/pi-apps/commit/fd584580618f627f6508f686a5a01b1b7cc67cfc

File: apps/Oh My Posh/install-64 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+wget -O /tmp/oh-my-posh https://github.com/JanDeDobbeleer/oh-my-posh/releases/latest/download/posh-linux-arm64
```
Comment (theofficialgman) [inline apps/Oh My Posh/install-64:3]: see above

File: etc/categories [hunk 3]
```diff
@@ -46,6 +46,8 @@ Deluge|Internet
 Descent 1|Games
 Descent 2|Games
 Deskreen|Internet
+Discord|Internet/Communication
+Discord PTB|Internet/Communication
```
Comment (theofficialgman) [inline etc/categories:50]: applications are added that are not part of this PR and Oh My Posh is not added in the categories file

Comment (libewa) [reply etc/categories:50]: rip switched up branches

File: etc/categories [hunk 4]
```diff
@@ -129,6 +129,7 @@ Node.js|Tools
 Notejot|Office
 Notepad ++|Programming
 OBS Studio|Multimedia
+Oh My Posh|Terminals/Appearance
```
Comment (theofficialgman) [inline etc/categories:132]: not a valid category
